contracts/asset-proxy/contracts/src/interfaces/IERC20Bridge.sol

```diff
@@ -21,7 +21,25 @@ pragma solidity ^0.5.9;
 
 contract IERC20Bridge {
 
-    // @dev Result of a successful bridge call.
+    /// @dev Emitted when a bridge transfer is completed.
+    /// @param bridge The bridge address.
+    /// @param fromToken The address of the "from" token.
+    /// @param toToken The address of the "to" token.
+    /// @param fromTokenAmount The "from" token amount consumed.
+    /// @param toTokenAmount The "to" token amount transferred.
+    /// @param from Supplier of "fromToken".
+    /// @param to Receiver of "toToken".
+    event ERC20BridgeTransfer(
+        address indexed bridge,
```

Comment:
Do we actually need to log the address of each bridge? Seems uncommon for events to do this.

Reply:
Oh, no, you're right. I forgot receipts will include the logger address.